scripts/build.mjs

```diff
@@ -0,0 +1,42 @@
+const buildPackages = async () => {
+  console.log('🏎️  Building packages...')
+  await $`vite build`
+}
+const buildTypes = async () => {
+  console.log('🏎️  Building types...')
+  await $`tsc -p tsconfig.build.json`
+}
+
+const concatAssets = async ({from, to}) => {
+  const files = await glob(from)
+  const filesContent = await Promise.all(files.map(file => fs.readFile(file)))
+  const result = filesContent.join('\n')
+  await fs.writeFile(to, result)
+  console.log(`${files.length} concatenated`)
+  console.log(to)
+}
+
+const buildPostcssExport = async () => {
+  console.log('🏎️  Building PostCSS export...')
+  await concatAssets({
+    from: ['packages/theme/base/breakpoints.css'],
+    to: 'dist/postcss.css'
+  })
+}
+
+const buildSCSSExport = async () => {
+  console.log('🏎️  Building SCSS export...')
+  await concatAssets({
+    from: ['packages/theme/base/breakpoints.scss'],
+    to: 'dist/scss.scss'
+  })
+}
+
+const main = async () => {
```

Comment:
that smells gulp

Reply:
can there be better solution? whether using vite or not

Reply:
There is rollup plugin https://www.npmjs.com/package/rollup-plugin-copy and there is vite plugin https://www.npmjs.com/package/vite-plugin-static-copy, rollup plugin is somehow popular, but it's unclear if this plugin is supported right now or will be supported in the future, there is also only one maintainer. So it feels like it's easier to implement similar plugin ourselves than getting yet another dependency.